patches/nan/nan_string_test_alignment.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: clavin <clavin@electronjs.org>
+Date: Wed, 12 May 2021 12:43:07 -0600
+Subject: nan_string_test_alignment
+
+Modifies a NAN test to avoid a debug check pertaining to efficient string alignment.
```

Comment:
1. Which DCHECK?
2. As above, under what circumstances is it safe to remove this patch?

Reply:
We should actually be able to upstream this fairly easily if the Nan maintainers agree to it! I'll add this to the patch 🙇‍♀️